### PR TITLE
Add Astarte Dashboard's documentation

### PR DESCRIPTION
**Pull request by davidebriani**
This PR adds a page in the User Documentation to document the usage of the Astarte Dashboard.

### CODE REVIEW

File: doc/pages/user/015-astarte_dashboard.md [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+## Introduction
+
+The Astarte Dashboard is a Single Page Application to get an overview of and manage your Realm on
+any browser. The Dashboard is designed to be a quick and easy way to give you immediate feedback on
+your work and as a quick and intuitive way to configure your realm. It is not meant to be operated
+by end-users, rather by infrastructure maintainers, owners who need information on the system
+status, and those working on projects based on Astarte. It is shipped by default with the Astarte
+standard distribution.
+
+The Dashboard is a graphical frontend for Astarte APIs; `astartectl`,
```
Comment (Pavinati) [inline doc/pages/user/015-astarte_dashboard.md:19]: The Dashboard is a graphical frontend for Astarte APIs, much like the CLI tool `astartectl`

File: doc/pages/user/010-interacting_with_astarte.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ When interacting with Astarte as a User, you have several options to choose from
 * [astartectl](https://github.com/astarte-platform/astartectl): `astartectl` is the main command-line tool to interact with Astarte clusters, which packs
   in a number of subcommands to interact with Astarte API sets. It is a swiss army knife to perform daily operations on Astarte Clusters, and it abstracts
   most Astarte API interactions in a user-friendly way.
-* Astarte Dashboard: Astarte provides a built-in UI that can be used for managing Interfaces, Devices and Triggers. If your cluster has the dashboard enabled, it can
-  be usually found at `dashboard.<base Astarte URL>`. It is meant to be a graphical, user-friendly tool to perform daily operations in the Cluster.
+* [Astarte Dashboard](015-astarte_dashboard.html): Astarte provides a built-in UI that can be used for managing Interfaces, Devices and Triggers.
+  It is meant to be a graphical, user-friendly tool to perform daily operations in the Cluster.
```
[the remark this answers is not in the document]
Comment (davidebriani) [reply doc/pages/user/010-interacting_with_astarte.md]: I agree, I changed to `daily operations on Realms`.

File: doc/pages/user/015-astarte_dashboard.md (hunk 1 above)
Comment (davidebriani) [reply doc/pages/user/015-astarte_dashboard.md]: I tried to rephrased, see if it's any better now.

File: doc/pages/user/015-astarte_dashboard.md [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+Upon successful login, the main screen is the home page that provides an overview of the realm
+status and resources.
+
+![Astarte Dashboard with a realm overview](assets/astarte-dashboard-realm-overview.png)
+
+### API Status
+
+The API Status gives you general information about the status of services:
+
+- **Realm Management** is an administrator-like API for configuring a Realm. It is used for managing
```
Comment (Pavinati) [inline doc/pages/user/015-astarte_dashboard.md:68]: Realm Management does not handle devices. It only manages the installation/retrieval of interfaces and triggers.

File: doc/pages/user/015-astarte_dashboard.md [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+- Ownership: the write-only allowed actor. All the other actors are read-only.
+- Mappings: a list of endpoints that represent the data structure, following REST controller
+  semantics.
+
+You can learn more about Interface definitions in the
+[documentation's reference](030-interface.html).
+
+Note that when creating interface drafts, or for testing purposes in general, it is recommended to
+use `0` as the major version: to prevent data loss, Astarte allows only interfaces with
+`major_version` == `0` to be deleted.
```
Comment (Pavinati) [inline doc/pages/user/015-astarte_dashboard.md:136]: in a text description i'd avoid equal sign to express "equals"

File: doc/pages/user/015-astarte_dashboard.md [hunk 5]
```diff
@@ -0,0 +1,394 @@
...
+### Editing interfaces
+
+From the Interface list, you can select an interface to load and view its details in the Interface
+Editor's page. Clicking on the interface's name will select the interface's latest revision;
+clicking on a specific major of the interface will select the latest revision for that major. Note
+that interface revisions follow the [Semantic Versioning](https://semver.org/) convention.
+
+Once the Interface Editor is loaded you can review, update or delete the interface's definition.
+
+Note that to prevent data loss, Astarte allows only interfaces with `major_version` == `0` to be
```
Comment (Pavinati) [inline doc/pages/user/015-astarte_dashboard.md:147]: same as previous comment

File: doc/pages/user/015-astarte_dashboard.md [hunk 6]
```diff
@@ -0,0 +1,394 @@
...
+- A `source`, containing the block's implementation, written in the
+  [DSL format](https://docs.astarte-platform.org/flow/snapshot/0010-defining-a-pipeline.html) for
+  Pipelines.
+
+Confirm the block's creation by clicking on the **Create new block** button.
+
+To learn more about blocks definition you can read the
+[documentation's reference](https://docs.astarte-platform.org/flow/snapshot/0003-blocks.html).
+
+### Editing blocks
```
Comment (Pavinati) [inline doc/pages/user/015-astarte_dashboard.md:316]: As for now blocks cannot be edited

File: doc/pages/user/015-astarte_dashboard.md [hunk 7]
```diff
@@ -0,0 +1,394 @@
...
+- `source`, containing the pipeline's implementation, written in the
+  [DSL format](https://docs.astarte-platform.org/flow/snapshot/0010-defining-a-pipeline.html) for
+  Pipelines.
+- `schema`, reporting the JSON Schema definition of how a configuration should look like for the
+  pipeline.
+- `description`, explaining the scope and supposed usage of the pipeline.
+
+Then hit the **Create new pipeline** button to confirm the pipeline's definition.
+
+### Editing pipelines
```
Comment (Pavinati) [inline doc/pages/user/015-astarte_dashboard.md:360]: Likewise for pipelines, they can't be edited

File: doc/pages/user/015-astarte_dashboard.md (hunk 3 above)
Comment (davidebriani) [reply doc/pages/user/015-astarte_dashboard.md:68]: Oh, I copied that line from https://docs.astarte-platform.org/1.0/020-components.html#realm-management
I guess we should change that too.

Comment (davidebriani) [reply doc/pages/user/015-astarte_dashboard.md]: Fixed them both.

File: doc/pages/user/015-astarte_dashboard.md (hunk 4 above)
Comment (davidebriani) [reply doc/pages/user/015-astarte_dashboard.md]: I agree, fixed it.

File: doc/pages/user/015-astarte_dashboard.md (hunk 6 above)
Comment (davidebriani) [reply doc/pages/user/015-astarte_dashboard.md]: You're right, I went with `managing blocks`

File: doc/pages/user/015-astarte_dashboard.md (hunk 7 above)
Comment (davidebriani) [reply doc/pages/user/015-astarte_dashboard.md]: Fixed it

**Review by Pavinati — APPROVED** [review]

**Review by bettio — APPROVED** [review]

**Pull request merged**